### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by deniurchak**

### CODE REVIEW

File: signing-service-challenge/README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
 
 ## Instructions
 
-This challenge is part of the SWE interview process at fiskaly.
+This challenge is part of the software engineering interview process at fiskaly.
 
 If you see this challenge, you've passed the first round of interviews and are now at the second and last stage.
 
-We would ask you to prepare the challenge below to have a basis for discussion in the skill fit interview with two colleagues from the development department.
+We would like you to complete the challenge below. The two colleagues from the development department will then discuss your solution with you in the skill fit interview.
```
Comment (auxdevelopment) [inline signing-service-challenge/README.md:9]: We don't ask applicants to complete the challenge if they don't find the time. Partial solutions are fine as well.

Comment (auxdevelopment) [reply signing-service-challenge/README.md:9]: The way it is now phrased sounds like you should know the "two colleagues from the development department"

File: signing-service-challenge/README.md [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
 
-`quality > quantity`
+The `quality` of your code is more important to us than the `quantity`.
```
Comment (auxdevelopment) [inline signing-service-challenge/README.md:11]: If we put it like this I would ditch the back ticks...

File: signing-service-challenge/README.md [hunk 3]
```diff
@@ -66,19 +66,19 @@ For now we need to provide two operations to our customers:
 
 Think of how to expose these operations through a RESTful HTTP based API.
 
-List / retrieval operations can optionally be implemented but aren't necessary by any means.
+`List / retrieval operations` can optionally be implemented but are not required.
 
 #### QA / Testing
 
-As we're in the area of compliance technology we need to make sure that our implementation is verifiably correct. Think of an automatable way to assure the correctness (in this challenge: adherence to the specifications) of the system.
+As we are in the business of compliance technology, we need to make sure that our implementation is verifiably compliant. Think of an automatable way to assure the correctness (in this challenge: adherence to the specifications) of the system.
```
Comment (auxdevelopment) [inline signing-service-challenge/README.md:73]: Tbh I think `correct` fits better here.

File: signing-service-challenge/README.md [hunk 4]
```diff
@@ -66,19 +66,19 @@ For now we need to provide two operations to our customers:
...
-- For now it is enough to store signature devices in memory. Efficiency is not a priority for this. We might want to scale out though, therefore keep in mind that we might want to switch to a relational database at some point when designing your storage logic.
+- For now it is enough to store signature devices in memory. Efficiency is not a priority for this. In the future we might want to scale out. As you design your storage logic, keep in mind that we may later want to switch to a relational database.
 
 #### Credits
 
-This challenge is heavily influenced by the KassenSichV (Germany) as well as the RKSV (Austria) and our solutions for them.
+This challenge is heavily influenced by the [`KassenSichV`](https://kassensichv.com/) (Germany) as well as the [`RKSV`](https://www.wko.at/branchen/information-consulting/unternehmensberatung-buchhaltung-informationstechnologie/it-dienstleistung/BGBLA-2015-II-410-EN.PDF) (Austria) and our [solutions](https://developer.fiskaly.com/api) for them.
```
Comment (auxdevelopment) [inline signing-service-challenge/README.md:84]: I refrained from putting the links there on purpose.

File: signing-service-challenge/README.md (hunk 1 above)
Comment (deniurchak) [reply signing-service-challenge/README.md:9]: changed to `attempt the challenge`. That fine?

File: signing-service-challenge/README.md (hunk 2 above)
Comment (deniurchak) [reply signing-service-challenge/README.md:11]: fixed

File: signing-service-challenge/README.md (hunk 3 above)
Comment (deniurchak) [reply signing-service-challenge/README.md:73]: reverted

File: signing-service-challenge/README.md (hunk 4 above)
Comment (deniurchak) [reply signing-service-challenge/README.md]: so that you know that the people can google?:)

**Pull request merged**